core/Access.php

```diff
@@ -170,18 +171,7 @@ public function reloadAccess(Auth $auth = null)
 
         // case the superUser is logged in
         if ($result->hasSuperUserAccess()) {
-            return $this->reloadAccessSuperUser();
-        }
-
-        // in case multiple calls to API using different tokens, we ensure we reset it as not SU
-        $this->setSuperUserAccess(false);
```

Comment:
`setSuperUserAccess(false)` is no longer needed since there was already a check for `hasSuperUserAccess()` so we can be sure the user won't have super user access. Removing this check works as we now do always reset the sites at the beginning of this method via `resetSites()`